src/test/java/service/AccountServiceTest.java

```diff
@@ -0,0 +1,119 @@
+package service;
+
+import model.Account;
+import model.enams.Gender;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AccountServiceTest {
+
+    @Test
+    void testFilterAccountsByBalance() {
+        List<Account> accounts = createTestAccounts();
+        AccountService accountService = new AccountService();
```

Comment:
every test has accountService  initialization, put it in @BeforeAll setup()/init()